lib/win-exception-handler/src/exception_handling.rs

```diff
@@ -16,7 +16,7 @@ extern "C" {
     #[link_name = "callProtected"]
     pub fn __call_protected(
         trampoline: Trampoline,
-        ctx: *mut Ctx,
+        env: Option<NonNull<Env>>,
```

Comment:
The C code doesn't handle the `Option` correctly I guess (it expects a pointer).